dt-login/login-endpoints.php

```diff
@@ -106,6 +106,9 @@ public function check_auth( WP_REST_Request $request ) {
         }
 
         if ( DT_Login_Methods::MOBILE === $login_method ) {
+            /* Make sure that any WP login tokens are cleared before logging in with JWT */
+            wp_logout();
```

Comment:
@squigglybob , should we add a `is logged in check` first?

Reply:
Nope, it works fine. I've added the check in